### PR TITLE
Add periods to committees

### PR DESCRIPTION
**Pull request by deiferni**
A period represents a timespan similar to a fiscal year.

  - Display the current period on committee overview.
  - Add a wizard to close the current and create its successor.
  - Add a wizard that creates an initial period upon committee creation.

Addresses the first part of #1065.

screen shot 2015-11-18 at 17 53 30

untitled

### CODE REVIEW

File: opengever/meeting/browser/committee.py [hunk 1]
```diff
@@ -16,7 +17,11 @@ class CommitteeOverview(grok.View, OpengeverTab):
 
     def boxes(self):
         items = [
-            [{'id': 'upcoming_meetings',
+            [{'id': 'period',
+              'label': _('label_period', default=u'Period'),
```
Comment (phgross) [inline opengever/meeting/browser/committee.py]: Evtl. _('label_current_period', default=u'Current Period') --> Aktuelle Periode

File: opengever/meeting/tests/test_period_query.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        self.committee = create(Builder('committee_model'))
+        self.period = create(Builder('period').having(committee=self.committee))
+
+    def test_active_returns_only_active(self):
+        create(Builder('period').having(workflow_state='closed',
+                                        committee=self.committee))
+
+        self.assertEqual([self.period], Period.query.active().all())
+
+    def test_by_committee_returns_periods_for_committee(self):
```
Comment (phgross) [inline opengever/meeting/tests/test_period_query.py]: `test_by_committee_returns_periods_for_committee_in_all_states`

File: opengever/meeting/browser/committeeforms.py [hunk 3]
```diff
@@ -1,18 +1,157 @@
...
+    Second part of a two-step wizard. Loads data from IIWizardStorage to
+    also create the committee from the first step after successful submission.
+
+
+    This form extends from dexterity's DefaultAddForm to create the plone
+    content-type easily.
+
+    Does not extend from dexterity.AddForm to avoid auto-registering this
+    form as add-form for committees, this would conflict with the form
+    from the first step. Instead configure portal_type manually.
```
Comment (phgross) [inline opengever/meeting/browser/committeeforms.py]: Nice :tada:

File: docs/HISTORY.txt [hunk 4]
```diff
@@ -5,6 +5,14 @@ Changelog
 4.6.0 (unreleased)
 ------------------
 
+- Add periods to committees. A period represents a timespan similar to a fiscal year.
```
Comment (phgross) [inline docs/HISTORY.txt]: `timespan` --> `time span` ?

**Comment by phgross** [comment]
Nur noch zwei drei Details ansonsten ready for :ship: -ing

File: docs/HISTORY.txt (hunk 4 above)
Comment (deiferni) [inline docs/HISTORY.txt]: no :wink: http://dictionary.reference.com/browse/timespan

Comment (phgross) [inline docs/HISTORY.txt]: Ah ok :+1: ! 

Da sind Sie sich die Übersetzer Apps selber nicht ganz einig (http://dict.leo.org/ende/index_de.html#/search=time%20span&searchLoc=0&resultOrder=basic&multiwordShowSingle=on) :grin:

**Pull request merged**